packages/workspace/src/utils/runtime-lint-utils.ts

```diff
@@ -133,25 +134,17 @@ export function isAbsoluteImportIntoAnotherProject(imp: string) {
 
 export function findProjectUsingImport(
```

Comment:
Maybe we should change it to accept the locator?

Otherwise, we are doing the sorting for every import during linting, which seems unnecessary.